alpine/3.18/8.2/Dockerfile

```diff
@@ -65,6 +65,9 @@ ARG PHP_EXTENSIONS="\
 
 ARG PHP_PECL_EXTENSIONS="\
     apcu \
+    opentelemetry \
```

Comment:
please move those to ADDITIONAL_PHP_PECL_EXTENSIONS section

Also we need to build images and test them locally.

Afterwards we need to get an approve from @alexanderM91 and after merge build will be triggered for docker hub.


Also please do the same for 8.1 version

Reply:
updated Dockerfile for all PHP version